filebeat/docs/load-balancing.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[load-balancing]]
+== Configuring Load Balancing
+
+//REVIEWERS: Which outputs support load balancing? I only see the loadbalance option described under Logstash and Redis. Are any other outputs supported?
+
+Filebeat provides configuration options that you can use to fine
+tune load balancing when sending events to multiple hosts. 
+
+To enable load balancing, you specify `load_balance: true` when you configure
+the output. For example:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+output.logstash:
+  hosts: ["localhost:5044", "localhost:5045"]
+  loadbalance: true
+-------------------------------------------------------------------------------
+
+Filebeat can perform load balancing in a few different modes:
+
+* **Send events to one host after another:**
++
+By default, when you configure Filebeat to send events to multiple hosts
+(`load_balance: true`), Filebeat will send the events to one host after
```

Comment:
s/load_balance/loadbalance/

plus: 
- `filebeat.publish_async: false`
- `filebeat.spool_size = output.X.bulk_max_size`

The host choosen for the next batch is kind of random.

This is not really load-balancing from client point of view, but can still balance out network usage somehow if multiple filebeat instances are connected to one cluster.

Better option might be to set `loadbalance: false`, though. In this mode filebeat chooses on logstash only by random + there is a chance different filebeat instances choosing different logstash instances in a logstash cluster still somehow balancing some load.